crates/driver_virtio/Cargo.toml

```diff
@@ -11,6 +11,7 @@ bus-mmio = []
 bus-pci = []
 block = ["driver_block"]
 net = ["driver_net"]
+display = ["driver_display"]
```

Comment:
This feature can be `gpu`, as it's used in the `driver_virtio` crate and virtio devices only include `virtio_gpu`.